packages/cli/src/lib/utils/color-utils.ts

```diff
@@ -0,0 +1,23 @@
+import chalk from 'chalk';
+
+export const customColors = {
+  header: chalk.bold.hex('#1CEBCF'),
+
+  green: chalk.green,
+  yellow: chalk.yellow,
+  red: chalk.red,
+  gray: chalk.gray,
+
+  error: chalk.bgHex('#F64D64').hex('#272C3A'),
+  cmd: chalk.blueBright.bold,
+};
+
+type availableColors = typeof customColors | typeof chalk;
+type augmentedChalk = typeof customColors & typeof chalk;
+
+export const color: typeof customColors & typeof chalk = new Proxy(chalk, {
+  get: (chalk, name: keyof availableColors) =>
+    customColors[name] ? customColors[name] : chalk[name],
+}) as augmentedChalk;
```

Comment:
I'm really not fond of proxies. They tend to obfuscate logs and were often a smell of hacks in my past experiences.
I understand you want to 'augment' chalk, but I think there's a lot of more maintainable design patterns to consider before going the proxy way.

Reply:
I understand. So far, `Proxy` is the only pattern that worked for me.
I tried simply augmenting the object like `{...chalk, ...customColors}` but it causes some run time errors.

If you think Proxy should absolutely be avoided, I can try figuring out a cleaner approach.